ospd/server.py

```diff
@@ -67,12 +71,16 @@ def write(self, data: bytes):
 
         while True:
             if b_end > len(data):
-                self.socket.send(data[b_start:])
+                try:
+                    self.socket.send(data[b_start:])
```

Comment:
With the try except block i find it a bit difficult to read. The break after the try/except block confused me :-)